dependencies/bdw-gc/meson.build

```diff
@@ -0,0 +1,22 @@
+# Boehm-gc Dependency File
+#============================================================================
+
+
+## FIXME !! Boehm is supposed to be an optional dependency
```

Comment:
Could this have been fixed by NixOS/nix#4947?

Reply:
:) Quite possibly!

Reply:
quite busy with irl things atm but ill look into it :)